docs/data/material/components/stack/stack.md

```diff
@@ -7,30 +7,47 @@ githubLabel: 'component: Stack'
 
 # Stack
 
-<p class="description">The Stack component manages layout of immediate children along the vertical or horizontal axis with optional spacing and/or dividers between each child.</p>
+<p class="description">Stack is a container component for arranging elements vertically or horizontally.</p>
+
+## Introduction
+
+The Stack component manages the layout of its immediate children along the vertical or horizontal axis, with optional spacing and dividers between each child.
+
+:::info
+Stack is ideal for one-dimensional layouts, while Grid is preferable when you need both vertical _and_ hortizontal arrangement.
+:::
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
-## Usage
+## Basics
 
-`Stack` is concerned with one-dimensional layouts, while [Grid](/material-ui/react-grid/) handles two-dimensional layouts. The default direction is `column` which stacks children vertically.
+```jsx
+import Stack from '@mui/joy/Stack';
```

Comment:
Fixed in https://github.com/mui/material-ui/commit/deb6b03806a79430ecddda3c5b8ce64604b3fe48